src/main/java/com/bazel_diff/TargetHashingClient.java

```diff
@@ -45,7 +46,7 @@
                 impactedTargets.add(entry.getKey());
             }
         }
-        return bazelClient.queryForImpactedTargets(impactedTargets, avoidQuery);
+        return bazelClient.queryForImpactedTargets(impactedTargets, avoidQuery, hashAllTargets);
```

Comment:
If I'm understanding things correctly, I think you can just return impactedTargets here if you hash all targets, but I haven't tested this out.

Reply:
We could, yet the user could potentially provide `avoidQuery`, we should still respect that value

Reply:
In that case would it make sense to return "bazelClient.query..." if there's an avoidQuery and impactedTargets if there's no avoidQuery?